tests/vm_test_util.go

```diff
@@ -44,14 +44,14 @@ func (t *VMTest) UnmarshalJSON(data []byte) error {
 }
 
 type vmJSON struct {
-	Env           stEnv                `json:"env"`
-	Exec          vmExec               `json:"exec"`
-	Logs          []stLog              `json:"logs"`
-	GasRemaining  *math.HexOrDecimal64 `json:"gas"`
-	Out           hexutil.Bytes        `json:"out"`
-	Pre           core.GenesisAlloc    `json:"pre"`
-	Post          core.GenesisAlloc    `json:"post"`
-	PostStateRoot common.Hash          `json:"postStateRoot"`
+	Env           stEnv                 `json:"env"`
+	Exec          vmExec                `json:"exec"`
+	Logs          common.UnprefixedHash `json:"logs"`
```

Comment:
I'll wait to see how many fail (just updates the test suite), and will disable only the failing tests (so we're not overly zealous and accidentally miss consensus issues).